pkg/controller/controller.go

```diff
@@ -634,8 +634,8 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 	go cont.apicConn.Run(stopCh)
 }
 
-func (cont *AciController) enableGlobalSync(stopCh <-chan struct{}) {
```

Comment:
Since we are touching this file, can we update this name to: snatGlobalInfoSync per comment in previous review?

Similarly, can you update some of the log messages below as we discussed?

Reply:
Done